src/main/java/green/fallingsand/Sand.java

```diff
@@ -0,0 +1,60 @@
+package green.fallingsand;
+
+public class Sand {
+    private int fieldX = 3;
+    private int fieldY = 3;
+    private int[][] field = new int[fieldX][fieldY];
+
+    /**
+     * @return the value in field
+     */
+    public int get(int x, int y) {
+        return field[y][x];
+    }
+
+    /**
+     * Sets the value in field to be 1
+     */
+    public void put(int x, int y) {
+        field[y][x] = 1;
+    }
+
+    /**
+     * Moves all sand down one square
+     */
+    public void fall() {
+        for(int y = 0; y < fieldY; y++) {
+            for(int x = 0; x < fieldX; x++) {
+                if(field[y][x] == 1) {
+                    boolean atBottom;
+                    do {
+                        atBottom = fallSingular(x, y);
+                    } while (!atBottom);
```

Comment:
This method is supposed to move all the sand down ONE square. 
Your code doesn't do that.